### DISCUSSION TITLE
How to run on Linux

### DISCUSSION THREAD

**Discussion by zainhm10** [moonD4rk/HackBrowserData#319] [Q&A] [answered]
I have installed the repository from the command line but I have no idea how to run it and target my Firefox browser. 

Any assistance would be greatly appreciated :)

**Comment by moonD4rk** [comment, marked as answer]
You can directly double-click to run
Or use `./hack-browser-data -vv` in terminal. Then check if there is a "results" folder in the current directory.

**Reply by zainhm10** [reply]
My bad, I had an older version of go installed, it works now:)
